### DISCUSSION TITLE
RTSPS port

### DISCUSSION THREAD

**Discussion by jcraffael** [bluenviron/mediamtx#867] [Questions]
Hello! Above all thank you for the project and tutorial which saved my day!

I just have one question: from the yml file we see that the default port for RTSPS is 8555 and I'd to ask why picking this port number for RTSPS? By searching a little bit online I know that for RTSP port 8554 is an alternate for 554 to use, but for RTSPS the standard port definded by IANA is 332 and 8555 is an undefined one.  So why 8555 instead of 322? 8555 is picked only for convienence since it follows closely 8554 for RTSP, or it is defined somewhere by some public authority?

Thank you!

**Comment by aler9** [comment]
Thanks for pointing that out, the default port has been changed to 8322.

322 can't be used since it requires root privileges.

[a comment by an automated account is not shown]
